packages/semantic-release-config/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@energyweb/semantic-release-config",
-  "version": "0.0.3",
+  "version": "1.0.0",
```

Comment:
Do you have to manually update this? Or is done automatically as a part of some command?

Reply:
I have to do it manually right now, bc I forgot to put the branch where there changes are committing https://github.com/energywebfoundation/shared-configs/blob/008397daa00ea50c8f7761becf8cbc3dffbfb204/.github/workflows/publish.yml#L40. So yesterday the was published with newer version but bump version commit wasn't pushed to master (https://github.com/energywebfoundation/shared-configs/runs/5321605074?check_suite_focus=true [DRY RUN]). I hope I fixed this and from now on everything should be done automatically by rush.